File: OOPS/implementation.java
class Car{
    int modelyear;
    String modelname;
    Car(String modelname,int modelyear){
        this.modelname=modelname;
        this.modelyear=modelyear;
    }
    void display(){
        System.out.println(this.modelname+this.modelyear);
    }
}
class Main{
    public static void main(String args[]){
        Car ford=new Car("forda2",34);
        ford.display();
    }
}

File: OOPS/implementation.py
class Car:
    count=0
    def __init__(self,modelname,year):
        self.modelname=modelname
        self.year=year
        Car.count+=1
    def display(self):
        print(self.modelname,self.year)

ford=Car("z1",2015)
ford.display()
mitusbo=Car("f2",2012)
print(Car.count)